linode_api4/linode_client.py

```diff
@@ -871,6 +871,213 @@ def users(self, *filters):
         """
         return self.client._get_and_filter(User, *filters)
 
+    def logins(self):
+        """
+        Returns a collection of successful logins for all users on the account during the last 90 days.
+
+        API Documentation: https://www.linode.com/docs/api/account/#user-logins-list-all
+
+        :returns: A list of Logins on this account.
+        :rtype: PaginatedList of Login
```

Comment:
Please discard that suggestion. Because I found we didn't strictly enforce it the doc... 
https://linode-api4.readthedocs.io/en/latest/linode_api4/login_client.html#linode_api4.LinodeLoginClient.finish_oauth
This is an example where we didn't use the strict Python type in this field...